test/controllers/api/resource_controller_test.rb

```diff
@@ -27,8 +28,27 @@ class Api::ResourceControllerTest < ActionDispatch::IntegrationTest
       })
   end
 
-  test 'describe resource name and version: get #index as json' do
-    get api_url(version: @api_namespace.version, api_namespace: @api_namespace.slug), as: :json
+  test 'describe resource name and version: get #index as json with no ahoy-visit being tracked when tracking is disabled' do
```

Comment:
@alis-khadka can you add a test for the index endpoint as well? This is what is erroring on `nikean.org`
``` txt
Request URL: https://nikean.org/api/1/story_platform-select-reason_tags
Request Method: GET
Status Code: 500 Internal Server Error
Remote Address: 3.99.30.108:443
Referrer Policy: strict-origin-when-cross-origin
```
<img width="862" alt="Screen Shot 2022-12-04 at 11 21 09 AM" src="https://user-images.githubusercontent.com/35935196/205502587-8cafbdf5-6317-45a1-920a-616583e4427b.png">

Reply:
<img width="1475" alt="Screen Shot 2022-12-04 at 10 28 13 PM" src="https://user-images.githubusercontent.com/25191509/205503886-4dadd3a6-2151-402f-a032-93891996ab56.png">
@donrestarone  Isn't this the index endpoint? This test case is hitting the index endpoint.

Reply:
@alis-khadka ah okay, as long as your tests are raising the same error without your patch in place we are good